src/theme/variables/material.js

```diff
@@ -2,8 +2,8 @@ import color from 'color';
 
 import { Platform, PixelRatio } from 'react-native';
 
-const deviceHeight = platform.deviceHeight;
-const deviceWidth = platform.deviceWidth;
+const deviceHeight = PLATFORM.deviceHeight;
```

Comment:
This would crash the app since "PLATFORM" is undefined here but this file in never used so I think It can be deleted